### PR TITLE
Moar shows

### PR DESCRIPTION
**Pull request by alloy**

### CODE REVIEW

File: lib/components/artist/shows/index.js [hunk 1]
```diff
@@ -50,7 +51,12 @@ export default Relay.createContainer(Shows, {
         current_shows: partner_shows(status: "running") {
           ${LargeShowsList.getFragment('shows')}
         }
-        ${SmallShowsList.getFragment('artist')}
+        upcoming_shows: partner_shows(status: "upcoming") {
+          ${LargeShowsList.getFragment('shows')}
+        }
+        past_shows: partner_shows(status: "closed", size: 20) {
```
Comment (sarahscott) [inline lib/components/artist/shows/index.js]: Nice. I think 20 past shows is very reasonable.

**Comment by sarahscott** [comment]
🍏

File: lib/components/artist/shows/small_list.js [hunk 2]
```diff
@@ -61,16 +61,14 @@ const styles = StyleSheet.create({
...
 export default Relay.createContainer(SmallList, {
   fragments: {
-    artist: () => Relay.QL`
-      fragment on Artist {
-        partner_shows(status: "closed") {
-          meta_image {
-            cropped(width: 75, height: 75) {
-              url
-            }
+    shows: () => Relay.QL`
```
Comment (orta) [inline lib/components/artist/shows/small_list.js]: what does the  "`" do?

Comment (alloy) [inline lib/components/artist/shows/small_list.js]: Well, I’m glad you asked. I thought it was just a Relay transpolar thing, but it turns out to be a ES6 string literal that allows interpolation 🎉 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Comment (orta) [inline lib/components/artist/shows/small_list.js]: nice, looks good

**Pull request merged**